test/apis/language_recognizer_test.go

```diff
@@ -35,7 +35,7 @@ func TestAnalyzeOnDjango(t *testing.T) {
 }
 
 func TestAnalyzeOnCSharp(t *testing.T) {
-	isLanguageInProject(t, "s2i-dotnetcore-ex", "c#", []string{}, []string{"net6.0"})
+	isLanguageInProject(t, "s2i-dotnetcore-ex", "c#", []string{}, []string{"net8.0"})
```

Comment:
Same goes for this one. Might be better to add an additional test case instead of replacing the net6.0